src/Controller/Admin/ActionController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Controller\Admin;
+
+use AlterPHP\EasyAdminExtensionBundle\Controller\EasyAdminController;
+use App\Entity\Conference;
+use Symfony\Component\HttpFoundation\Response;
+
+class ActionController extends EasyAdminController
```

Comment:
```suggestion
class ConferenceActionController extends EasyAdminController
```

Reply:
Why **action**?

Reply:
Action was probably not really an appropriate naming, but I don't think Conference is a good naming either because it's a bit confusing as we are using entity related in other places in the admin.

Maybe something like AdminController would be better ?

Reply:
I updated it, it's now still the ConferenceController but it is no longer the main EasyAdmin Controller